.github/actions/update-dockerhub-description/action.yml

```diff
@@ -0,0 +1,35 @@
+name: 'Update Docker Hub README'
+description: 'Composite action to update Docker Hub README for OSS and EE images'
+inputs:
+  docker_username:
+    description: 'Docker username'
+    required: true
+  docker_password:
+    description: 'Docker password'
+    required: true
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Generate Docker Hub Description
+      shell: bash
+      run: |
+        .github/scripts/generate-docker-hub-description.sh
+
+    - name: Update Docker Hub Description for OSS image
+      uses: peter-evans/dockerhub-description@v4
```

Comment:
I prefer a commit hash here.
Even the Docker documentation uses the hash in their sample: https://docs.docker.com/build/ci/github-actions/update-dockerhub-desc/